src/app/projects/create-new-project/create-new-project.component.ts

```diff
@@ -77,6 +79,7 @@ export class CreateNewProjectComponent implements OnInit, AfterViewInit {
     }
 
     ngAfterViewInit() {
+        this.mapService.getMapViewOnEdit(37.97404469468311, 23.71933726268805);
```

Comment:
Why these numbers? 🤓

Reply:
These coordinates were used on old Google Maps as default location when user tries to create new project.